resources/migrations/20180130164001-issues-archived.down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE issues_archived;
```

Comment:
Seems safer to rename it and leave it be for a while, then do cleanup task once it is 100% it works well in both test and production.

Reply:
This is just a migration down, so why is the alter table really needed? We are not migrating down in production.

Reply:
I'm not decided on this. Suggest to keep it as is, might help during testing on develop.

Reply:
It's just because this breaks migration semantics I believe. To me migrations should be able to run up and down several times, and this would break that.

Reply:
Got you. Sounds reasonable. Changed to DROP.